manifest/openshift4/coreos/ncp-openshift4.yaml

```diff
@@ -753,15 +753,6 @@ spec:
           # we move the usr kmod files to this dir temporarily before
           # installing new OVS kmod and/or backing up existing OVS kmod backup
             mountPath: /tmp/nsx_usr_ovs_kmod_backup
-
-
-          # mount to which an OVS kmod file is copied.
-          - name: host-usr-share
-            mountPath: /host/usr/share
-          # mount host's depmod to create module dependencies after creating
-          # new OVS kmod files
-          - name: host-depmod
-            mountPath: /sbin/depmod
 {{else}}
```

Comment:
we can remove code line #L756 - 767.
{{else}}
..
#mountPath: /tmp/nsx_usr_ovs_kmod_backup